cmd/oras/internal/errors/errors.go

```diff
@@ -62,10 +62,18 @@ func CheckArgs(checker func(args []string) (bool, string), Usage string) cobra.P
 
 // NewErrEmptyTagOrDigest creates a new error based on the reference string.
 func NewErrEmptyTagOrDigest(ref registry.Reference) error {
-	return NewErrEmptyTagOrDigestStr(ref.String())
+	return NewErrEmptyTagOrDigestStr(ref.String(), nil, false)
```

Comment:
Is `NewErrEmptyTagOrDigest` still in use? If not we can remove it.

Reply:
No, I was thinking about removing it.

Reply:
removed this function